shared-data/contrib/hints/hints.py

```diff
@@ -68,6 +68,13 @@ class hintsCommand(Command):
             _('Learn how to get the most out of Mailpile\'s spam filter'),
             '/page/hints/spam.html'),
 
+	# Show the user how to organize their sidebar after 6 days
+	# Remind them about it every 90 days in case they forgot how
+	# FIXME: Allow the user to say, "I know, shut up forever please."
+	('organize-sidebar', 6, 90,
```

Comment:
Let's just never remind the user about this one, I don't think it's necessary. Setting the interval to 99999 should do the trick. :-)